src/app/components/dashboard/dashboard.component.ts

```diff
@@ -223,38 +223,29 @@ export class DashboardComponent implements OnInit {
 			this.smoothingClaimed = this.data.rocketpool.smoothingPoolClaimed.dividedBy(new BigNumber('1e9'))
 			this.smoothingUnclaimed = this.data.rocketpool.smoothingPoolUnclaimed.dividedBy(new BigNumber('1e9'))
 			this.unclaimedRpl = this.data.rocketpool.rplUnclaimed
-		} catch (e) {}
+		} catch (e) {
+			console.warn('can not update smoothing pool', e)
```

Comment:
Should be "cannot". Has several occurrences.

Reply:
fixed